File: 0217-contains-duplicate/0217-contains-duplicate.java
class Solution {
    public boolean containsDuplicate(int[] nums) {
        Set<Integer> m = new HashSet();
        for (int i = 0; i < nums.length; i++)
            if (m.contains(nums[i]))
                return true;
            else
                m.add(nums[i]);
        return false;
    }
}

File: 0217-contains-duplicate/0217-contains-duplicate.js
/**
 * @param {number[]} nums
 * @return {boolean}
 */
var containsDuplicate = function(nums) {
    const s = new Set()
    for (n of nums ){
        if (s.has(n)) return true
        else s.add(n)
    }
    return false
};